roles/sap_vm_provision/defaults/main.yml

```diff
@@ -578,8 +580,106 @@ sap_vm_provision_kubevirt_vm_host_specifications_dictionary:
           mountpoint: /data0
           disk_count: 1                  # default: 1
           disk_size: 512                 # size in GB, integer
-          disk_type: nas                 # KubeVirt Storage Clas
+          disk_type: nas                 # KubeVirt Storage Class
 
+############################################
+# Red Hat OpenShift Virtualization         #
+############################################
+
+# Namespace where the VM should be created in
+sap_vm_provision_ocp_target_namespace: sap
+
+# Username to be created on guest
+sap_vm_provision_ocp_os_user: cloud-user
+
+# Password for the above user
+sap_vm_provision_ocp_os_user_password: ""
```

Comment:
Same as above, can we make it fail if value is empty?

Reply:
I could add a check addressing this.

Reply:
+1

Reply:
Check added